waywiser_command_control/config/command_control.yaml

```diff
@@ -13,3 +13,61 @@ onboard_twist_mux:
         topic: waywise_vel
         timeout: 0.5
         priority: 60
+
+collision_monitor:
```

Comment:
If this is based on an existing external file, I would like to see a comment here referencing it. Is this tuned to our setup in any way? A link to documentation, e.g., on how to adapt / fine tune this to different vehicles would be great.

Reply:
The polygon shapes are tuned for the RC car. Unfortunately I didn't find any documentation for fine tuning the polygons for humble release. There exists configuration guide from nav2 but it refers to a later release and contains more parameters than the humble release. I believe that "https://github.com/ros-planning/navigation2/tree/humble/nav2_collision_monitor" is the most accurate link for documentation. I added it in the config file and in the package.xml.